### ISSUE TITLE
Create Study Cards with Key Texts

### ISSUE THREAD

**Issue by LeonelRFF** [ECuiDev/obsidian-quiz-generator#22]
Hi, is it possible to create cards with short fragments?
Use case: When studying, in notes, PDFs or with the plugin [Copilot](https://github.com/logancyang/obsidian-copilot) I come across a phrase, fragment or sentence that perfectly explains a topic, that 'Aha!' or 'Eureka!' moment, as Archimedes would say.
Implementation could take the following forms:

- Using selected text
- Using clipboard content
- Upon opening the plugin interface, having the option to write manually

I provide these alternatives for diversity, as users may prefer different methods. Personally, I think I would use one of these options.
Greetings.

**Comment by ECuiDev** [comment]
Do you mean instead of selecting notes, you can directly type/paste the content you want to generate questions from?

**Comment by LeonelRFF** [comment]
Correct, when I read a PDF and there is something I don't understand, I ask the AI ​​to explain it in a very simple way. I have created prompts specifically with the idea of ​​converting it into flashcards but it often gets confused, forgets the syntax, sometimes changes it and other inconveniences, now, after trying this plugin, it is more efficient for this.

**Comment by LeonelRFF** [comment]
I have seen that this could have a drawback that already occurs, when trying to do it with a note that contained three paragraphs the AI ​​hallucinated and invented things that had nothing to do with it, I reduced the number of questions I had to ask but I don't know if it is completely safe, perhaps it would be better for the AI ​​to create only what is fair and necessary, since I don't know at what point it will mix what is invented with what is real, another option would be to be able to lower the temperature to reduce this even more.
